server/server/src/static_files.rs

```diff
@@ -44,3 +50,21 @@ async fn files(
 
     Ok(response)
 }
+
+async fn plugins(req: HttpRequest, settings: Data<Settings>) -> Result<HttpResponse, Error> {
+    let path = req.match_info();
+    let plugin = path
+        .get("plugin")
+        .ok_or(std::io::Error::new(ErrorKind::NotFound, "Plugin not found"))?;
```

Comment:
I think this can be unwrapped ? Can also use something similar to serve_frontend:
https://github.com/openmsupply/open-msupply/blob/80609f9178e7ccc06923b18454f951cd1e1d9d61/server/server/src/serve_frontend.rs#L27-L29

As per: https://actix.rs/docs/extractors#path, would be simpler without filename (which i believe is not needed ?)